Solution1/BusinessLayer/DTOs/WeatherDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.DTOs
+{
+    public class WeatherDTO
+    {
+        public string CityName { get; set; }
+        public double Temp { get; set; }
+        public string Comment { get; set; }
+
+        public override string ToString()
```

Comment:
single responsibility principle - dto can't be responsible for printing weather. think where can you place this logic (we used it previously )

Reply:
+